src/components/ec-loading/ec-loading.vue

```diff
@@ -31,6 +37,15 @@ export default {
       type: Number,
       default: 48,
     },
+    transparent: {
+      type: Boolean,
+      default: true,
+    },
+  },
+  computed: {
+    isTransparent() {
+      return this.transparent || (!this.show && !this.transparent);
```

Comment:
This could be simplified to `this.transparent || !this.show`.  Try it yourself: https://www.dcode.fr/boolean-expressions-calculator.